### PR TITLE
Dev/pyctbgui merge

### PR DESCRIPTION
**Pull request by thattil**
- Moved pyctbgui repos into here with all the commits
- Removed old ctbgui
- modified cmakelists  and cmk.sh to point to the new pyctbgui
TODO (@erikfrojdh ):
- create cmake file for pyctbgui insted of makefile (also install when SLS_INSTALL_PYTHONEXT is enabled)

### CODE REVIEW

File: pyctbgui/.clang-format
Comment (erikfrojdh) [inline pyctbgui/.clang-format:1]: I guess we don't need a separate .clang-format for the pyctbgui?

File: pyctbgui/.github/ISSUE_TEMPLATE/bug_report.md
Comment (erikfrojdh) [inline pyctbgui/.github/ISSUE_TEMPLATE/bug_report.md:1]: Remove file

File: pyctbgui/.github/ISSUE_TEMPLATE/feature_request.md
Comment (erikfrojdh) [inline pyctbgui/.github/ISSUE_TEMPLATE/feature_request.md:1]: Remove file

**Review by erikfrojdh — COMMENTED** [review]
W should have a look a the files in pyctbgui/.github before merging some files can be removed.

File: pyctbgui/.github/workflows/build-and-test-inplace.yml
Comment (thattil) [reply pyctbgui/.github/workflows/build-and-test-inplace.yml:1]: ok i will leave this folder

**Review by erikfrojdh — APPROVED** [review]
Looks good. Let's merge it and then modify as needed later

**Pull request merged**
